.github/workflows/ci-prb.yml

```diff
@@ -31,11 +31,18 @@ jobs:
         with:
           minimum-size: 8GB
           maximum-size: 16GB
-      - name: Build and Test
+      - name: Build and Test (Linux)
+        if: runner.os == 'Linux'
         env:
           CI: true
           JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8
-        run: ./gradlew --parallel clean test
+        run:  sudo env "PATH=$PATH" bash -c "ulimit -l 65536 && ulimit -a && ./gradlew --no-daemon --parallel clean test"
```

Comment:
Is it required to execute `ulimit` in the same `bash -c` command as `./gradlew`?
I was thinking about a separate optional step, something like:
```yaml
- name: Configure Linux for io_uring
  if: runner.os == 'Linux'
  run: sudo ulimit -l 65536 && ulimit -a
```
That way there is no repetition for the gradle command and tests won't run with root privileges.

Reply:
`ulimit` is discussed here:  https://github.com/actions/virtual-environments/issues/4683

Reply:
Thanks for the context!